### PR TITLE
[TRAFODION-2744] Correct the Descriptions and Example of *parallel=num (Copy Operators)* in *Trafodion ODB User Guide*

### PR DESCRIPTION
**Pull request by liuyu000**

### CODE REVIEW

File: docs/odb_user/src/asciidoc/_chapters/load.adoc [hunk 1]
```diff
@@ -1080,24 +1080,54 @@ define the size of this buffer in two different ways: +
...
+
+&#8226; To extract data from partitioned source table. +
+The number of the threads is as many as the parallel argument. +
+
+&#8226; To write data to the target table. +
+The number of the threads is equal to *parallel argument* * *number of loaders*. +                                                                                                        
+
+*Example* 
+
+If you specify parallel argument = 4 and the source table is made of 32 partitions, then odb start: +
```
Comment (DaveBirdsall) [inline docs/odb_user/src/asciidoc/_chapters/load.adoc]: Grammar: Consider "If you specify parallel argument = 4 and the source table has 32 partitions, then odb starts: +"

File: docs/odb_user/src/asciidoc/_chapters/load.adoc [hunk 2]
```diff
@@ -117,7 +117,7 @@ If you add a `+` sign in front of the file-name, odb  *appends* to `<file>`inste
  +
 - one thread to read from the input file and +
 - as many threads as the parallel argument to write via ODBC. This option is database independent.
-| `errmax=num` | odb prints up to num error messages per rowset. Normally used with soe to limit the number of error messages printed to the standard error
+| `errmax=num` | odb prints up to num error messages per rowset. Normally used with soe to limit the number of error messages printed to the standard error.
```
Comment (DaveBirdsall) [inline docs/odb_user/src/asciidoc/_chapters/load.adoc]: Perhaps "... standard error stream."

File: docs/odb_user/src/asciidoc/_chapters/load.adoc (hunk 1 above)
Comment (liuyu000) [reply docs/odb_user/src/asciidoc/_chapters/load.adoc]: Thanks Dave.
I've corrected. :)

File: docs/odb_user/src/asciidoc/_chapters/load.adoc (hunk 2 above)
Comment (liuyu000) [reply docs/odb_user/src/asciidoc/_chapters/load.adoc]: Thanks Dave.
I've corrected. :)

**Pull request merged**